posts/Yoann/2018-05-29-hibernate-search-5-10-1-Final-5-9-2-Final-5-6-5-Final.adoc

```diff
@@ -0,0 +1,57 @@
+= Triple bugfix release for Hibernate Search
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published bugfix releases for the branches currently in maintenance mode:
```

Comment:
Maybe "the branches we actively maintain"?

Reply:
Will do